test/reduce/remove_unreferenced_instruction_test.cpp

```diff
@@ -83,149 +87,14 @@ TEST(RemoveUnreferencedInstructionReductionPassTest, RemoveStores) {
   ASSERT_TRUE(ops[1]->PreconditionHolds());
   ops[1]->TryToApply();
 
-  CheckEqual(env, expected, context.get());
-}
-
-TEST(RemoveUnreferencedInstructionReductionPassTest, ApplyReduction) {
```

Comment:
This test was essentially testing the reducer algorithm, not the remove unreferenced instruction pass, so I removed it. I extended the test above slightly too, which is basically doing the same thing without testing the reducer algorithm.

Reply:
Thanks.